web-ui/src/main/resources/catalog/locales/en-admin.json

```diff
@@ -890,6 +890,9 @@
     "system/site/siteId-help": "Unique catalogue identifier. After changing the site identifier, user should update the index and the logo from the administration panel.",
     "system/site/svnUuid": "SVN UUID",
     "system/site/svnUuid-help": "Subversion repository unique identifier",
+    "metadata/vcs": "Version Control System (VCS)",
+    "metadata/vcs/enable": "Enable VCS",
+    "metadata/vcs/enable-help": "VCS is using SVN tool to follow metadata changes. This feature is experimental. It is also not operational on NFS filesystem. Once enabled, the application needs a restart.",
```

Comment:
I would update a bit the text to indicate: Once enabled **or disabled**, ....

Reply:
Message updated @josegar74